### PR TITLE
Remove mathn dependency

### PR DESCRIPTION
**Pull request by netizer**
As `mathn` is deprecated and as it overwrites basic math operations globally (which breaks applications written in ruby 5.1), I prepared this PR that removes `mathn` and relies on standard ruby methods. 

The tests for this PR break for Ruby 2.0. 
Please advise, if I should fix the tests for this version of ruby, or it's better to remove the support for version 2.0 (which is pretty old)

### CODE REVIEW

**Comment by a-fent** [comment]
Polite bump :) - would it be possible to integrate this patch please? Ruby 2.5 is fairly widespread now and the mathn dependency creates an awkward global change.

**Comment by bfoz** [comment]
Thanks for the bump, and sorry for disappearing again.

I'm fine with dropping support for ruby 2.0. It looks like 2.1 and 2.2 are also EOL, so there's no need to test on those either. I'll push a commit to update the gemspec, etc.

Would you mind cleaning up the whitespace changes in the commit? I know the files have a lot of trailing whitespace garbage, but I prefer to keep the commits clean. Fixing whitespace on a line that's otherwise changed is fine though.

File: lib/geometry/annulus.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def outer_diameter
 	# @!attribute outer_radius
 	#   @return [Number]  the outer radius
 	def outer_radius
-	    @outer_radius || (@outer_diameter && @outer_diameter/2)
+	    @outer_radius || ((@outer_diameter && @outer_diameter).to_r/2)
```
Comment (bfoz) [inline lib/geometry/annulus.rb]: It looks like an extra set of parenthesis was added here

**Comment by netizer** [comment]
@bfoz done. Please, let me know if I can do something else.

**Comment by bfoz** [comment]
@netizer Thanks for taking care of this.

BTW, I updated the Travis config and it looks like we're good on rubies 2.5 and 2.6 now

**Comment by netizer** [comment]
Awesome! Thanks for the merge :-)

**Pull request merged**